Apps/Playground/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 set(BABYLONSCRIPTS
     "../BabylonScripts/babylon.glTF2FileLoader.js"
     "../BabylonScripts/babylon.max.js"
+    "../BabylonScripts/babylon.max.js.map"
```

Comment:
Is this generated with normal babylon.max.js compilation? Is there any way through github actions to test that babylon.max.js.map matches the babylon.max.js file checked in?

Reply:
I'm not sure there's an easy way to test if the sourcemap matches the .js file. If we start consuming babylon.js via npm then we can just use the sourcemap file next to the .js file from inside the npm module...

I guess for now we could have @bghgary modify the CODEOWNERS file to include the .js.map file once this goes in, so we have a manual check to make sure both are updated together.

Reply:
The CODEOWNERS file covers all the files in the BabylonScripts dir. It should already cover them.